SI/Sem.04/Pract.04/Task13.cpp

```diff
@@ -0,0 +1,50 @@
+#include <iostream>
+using namespace std;
+
+bool checkPrime(int input) {
+	bool result = true;
```

Comment:
Either remove result or bool result = false(if you choose the 2nd , the next changes are needed).